blockservice/blockservice.go

```diff
@@ -64,41 +64,61 @@ type BlockService interface {
 	DeleteBlock(ctx context.Context, o cid.Cid) error
 }
 
+// BoundedBlockService is a Blockservice bounded via strict multihash Allowlist.
+type BoundedBlockService interface {
+	BlockService
+
+	Allowlist() verifcid.Allowlist
+}
```

Comment:
Mainly to forward Allowlist to Sessions. Can privatize it or anonymize

Reply:
I would embed `verifcid.AllowList` in the `BoundedBlockService` but this is also fine, up to you.

Reply:
I like the idea, so pushed a change